tests/unit/local/apigw/test_local_apigw_service.py

```diff
@@ -858,6 +858,20 @@ def test_lambda_auth_unauthorized_response(
         result = self.api_service._request_handler()
         self.assertEqual(result, unauth_mock)
 
+    @patch.object(LocalApigwService, "_invoke_lambda_function")
+    @patch.object(LocalApigwService, "_create_method_arn")
+    def test_lambda_authorizer_pass_context_http(self, method_arn_mock, mock_invoke):
+        mock_get_context = Mock()
+        route_event = {}
+
+        auth = LambdaAuthorizer(Mock(), Mock(), "auth_lambda", [], Mock(), Mock(), Mock())
+        auth.is_valid_response = Mock(return_value=True)
+        auth.get_context = Mock(return_value=mock_get_context)
+        self.http_v2_payload_route.authorizer_object = auth
+
+        self.http_service._invoke_parse_lambda_authorizer(auth, {}, route_event, self.http_v2_payload_route)
```

Comment:
I am not able to find a test case that cover the V1 route. Is there a test case that cover this case?

Reply:
There doesn't appear to be a case for the V1 route, thanks for catching that